### PR TITLE
Release/v0.13.0

### LINKED ISSUE #79: Simplify variable namespace
It can be confusing/misleading when the naming convention is not consistent. This results in confusion with the different types for variables. I suggest renaming these variable namespaces associated with reading or writing tables.

Generally, a filename can be referred to as `<something>_file` rather than `<something>`﻿and the corresponding table as `<something>_tab` rather than `<something>`.

Here's an example in `manual_bin_grid`:
``` python
    bin_info = join(fitspath, filename_dict['bin_info'])
    individual_info = join(fitspath, filename_dict['individual_prop'])
    pdf_pages = PdfPages(join(fitspath, 'manual_binning_shading.pdf'))

    tab = asc.read(bin_info)
    idv_tab = asc.read(individual_info)
```

List of changes:
- [x] `pdf_pages` -> `pp`
- [x] `out_pdf`/`pdf_pages` -> `pdf_file`
- [x] Use `_file` suffix when referring to filenames for tables
- [x] Use `_tab` suffix when referring to astropy `Table` objects or astropy `fits` objects

### LINKED ISSUE #94: Add plot differences function to LAC
Want to make a plot difference function for the LAC code much like the one used in the GPC code.

### PR DESCRIPTION
**Pull request by Reagen**
Includes: 
Closes #79 
Closes #94

### CODE REVIEW

File: analysis/deep2_r23_o32/binning/single_grid_r23.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
-def single_grid_r23(pdf_pages, npz_outfile, R23, O32, galinbin, log=None):
+def single_grid_r23(pdf_file, npz_outfile, R23, O32, galinbin, log=None):
     """
     This file holds the function to bin data for one dimensional binning
     along R23.
     Note: Not used in current analysis
 
-    :param pdf_pages: str. name of outputted pdf file
+    :param pdf_file: str. name of outputted pdf file
                       fitspath + name_dict['gridpdf_suffix']
```
Comment (astrochun) [inline analysis/deep2_r23_o32/binning/single_grid_r23.py:15]:
```suggestion
                     fitspath + name_dict['gridpdf_suffix']
```

File: analysis/deep2_r23_o32/calibration_plots.py [hunk 2]
```diff
@@ -47,132 +49,144 @@ def lac_gpc_plots(fitspath, fitspath_ini, dataset, raw=False,
...
         label = ['Detection', 'Robust Limits', 'DEEP2', 'MACT']
+        IDs = [det_ID, rlimit_ID, DEEP2_id, MACT_ID]
 
-    local_analog_calibration.main(lR23, lO32, OH, LAC_out_pdf, yra=[7.0, 9.0],
+    local_analog_calibration.main(lR23, lO32, OH, lac_pdf_file,
+                                  lac_diff_R23_pdf_file, lac_diff_O32_pdf_file,
+                                  ID=IDs, yra=[7.0, 9.0],
                                   ctype=c_var, label=label, marker=marker,
-                                  log=log)
+                                  log=None)
```
Comment (astrochun) [inline analysis/deep2_r23_o32/calibration_plots.py:176]: Why is this `log=None`?

File: analysis/deep2_r23_o32/plotting/color_bin_plotting.py [hunk 3]
```diff
@@ -50,23 +51,24 @@ def color_for_bin(fitspath, bin_info, pdf_file):
...
     """
     Plotting R23 vs O32 with a color map for
     metallicity and then for temperature
 
     :param fitspath: str. path where files are called from and saved to
-    :param bin_info: str. table created by binning code
-    :param temp_tab: str. table holding metallicity and
+    :param bin_info_file: str. table created by binning code
+    :param bin_derived_prop_file: str. table holding metallicity and
                      temperature measurements
```
Comment (astrochun) [inline analysis/deep2_r23_o32/plotting/color_bin_plotting.py:67]:
```suggestion
                                  temperature measurements
```

File: analysis/deep2_r23_o32/plotting/histogram_plots.py [hunk 4]
```diff
@@ -230,25 +230,25 @@ def run_histogram_fr(path, table_path, dict_list, verification_table,
...
                         outputted pdf_file will be saved
-    :param table_path: str. location of the flux_ratio table outputted by
+    :param input_file: str. location of the flux_ratio table outputted by
                             the R_temp_cal functions; can also be
                             combine_flux_table created zoom_and_gauss_general
     :param dict_list: ls. list of dictionaries whose data we want to plot
                             in a histogram
-    :param verification_table: str. table created independently of this code
+    :param valid_file: str. table created independently of this code
                         that tells us whether or not each bin is a detection
```
Comment (astrochun) [inline analysis/deep2_r23_o32/plotting/histogram_plots.py:239]:
```suggestion
                       that tells us whether or not each bin is a detection
```

File: analysis/deep2_r23_o32/stackboth_mastergrid.py [hunk 5]
```diff
@@ -82,13 +82,13 @@ def master_stacking(fitspath, fitspath_ini, dataset, grid_data_file,
         image2DM = np.ma.masked_array(image2DM, maskM[det3])
         log.debug("Mask[det3]", maskM[det3])
 
-    outfile = join(fitspath, filename_dict['comp_spec'])
-    if not exists(outfile):
-        stack_2d = np.zeros((len(R23_minimum) * len(O32_minimum), len(wave)),
+    stack_2d_file = join(fitspath, filename_dict['comp_spec'])
+    if not exists(stack_2d_file):
+        stack_2d_tab = np.zeros((len(R23_minimum) * len(O32_minimum), len(wave)),
                             dtype=np.float64)
```
Comment (astrochun) [inline analysis/deep2_r23_o32/stackboth_mastergrid.py:88]: Fix indentation here.

File: analysis/deep2_r23_o32/zoom_and_gauss_general.py [hunk 6]
```diff
@@ -535,23 +537,23 @@ def zm_general(dataset, fitspath, y_correction, thesis=False, log=None):
             lineflag[idx] = 1
 
     for ii in range(len(lambda0)):
-        out_pdf = join(fitspath, f"Zoomed_Gauss_{line_name[ii]}.pdf")
-        log.info(f"out_pdf: {out_pdf}")
+        pdf_file = join(fitspath, f"Zoomed_Gauss_{line_name[ii]}.pdf")
+        log.info(f"pp: {pdf_file}")
```
Comment (astrochun) [inline analysis/deep2_r23_o32/zoom_and_gauss_general.py:541]:
```suggestion
        log.info(f"pdf_file: {pdf_file}")
```

File: analysis/local_analog_calibration.py [hunk 7]
```diff
@@ -21,22 +21,158 @@ def bian18_R23(OH):
...
+def plot_differences(lR23_lO32, OH, pdf_file, data_input, data_err=[],
+                     OH_err=[], OH_range=[], data_range=[], marker=[], label=[],
+                     IDs=[], log=None):
+    """
+    Plot differences between LACR23 vs observed R23 as
+    a function of metallicity
+    Plot differences between LACO32 vs observed O32 as
+    a function of metallicity
+    Used by main() function
+    """
```
Comment (astrochun) [inline analysis/local_analog_calibration.py:53]: Why is this docstrings block so narrow?

**Review by astrochun — CHANGES_REQUESTED** [review]
See suggestions.

File: analysis/deep2_r23_o32/calibration_plots.py (hunk 2 above)
Comment (Reagen) [reply analysis/deep2_r23_o32/calibration_plots.py:176]: fixed

File: analysis/deep2_r23_o32/stackboth_mastergrid.py (hunk 5 above)
Comment (Reagen) [reply analysis/deep2_r23_o32/stackboth_mastergrid.py:88]: please check to make sure I fixed the problem you spotted

File: analysis/local_analog_calibration.py (hunk 7 above)
Comment (Reagen) [reply analysis/local_analog_calibration.py:53]: fixed

**Review by Reagen — COMMENTED** [review]
Test run completed

**Review by astrochun — APPROVED** [review]

**Pull request merged**